wallet/src/main/java/com/dimension/maskbook/wallet/ui/scenes/wallets/send/SendTokenHost.kt

```diff
@@ -36,31 +45,35 @@ import kotlin.time.ExperimentalTime
 @OptIn(ExperimentalMaterialNavigationApi::class, ExperimentalTime::class)
 @Composable
 fun SendTokenHost(
-    initialTokenData: TokenData,
+    tokenAddress: String,
+    onBack: () -> Unit,
     onDone: () -> Unit,
 ) {
-    val rootNavController = LocalRootNavController.current
     val context = LocalContext.current
+
     val navController = rememberNavController()
     val bottomSheetNavigator = rememberBottomSheetNavigator()
-    navController.navigatorProvider += bottomSheetNavigator
+    SideEffect {
+        navController.navigatorProvider += bottomSheetNavigator
```

Comment:
According to https://google.github.io/accompanist/navigation-material/ , we can use `val navController = rememberNavController(bottomSheetNavigator)` now